src/coreclr/vm/precode.cpp

```diff
@@ -662,4 +662,55 @@ BOOL DoesSlotCallPrestub(PCODE pCode)
     return FALSE;
 }
 
+PrecodeMachineDescriptor g_PrecodeMachineDescriptor;
+
+void PrecodeMachineDescriptor::Init()
```

Comment:
Why is there an Init routine as well as a set of non-static data member initializers here?

Reply:
I need to delete the initializers, that was an idea that didn't work out

Reply:
Are you doing this before you want this checked in? Given the snap deadline, I think we should probably miss it, and do a backport of the data descriptors when we're ready in reality to declare that .NET 9 might actually be useable with the cDAC and CLRMA. In general, I'm leery of stuff that has a static initializer, and that's what we have here now, and I don't see us fixing this before the snap happens.

Reply:
Yea, let's miss the snap. I can't fix and validate this in the next couple hours